src/components/Records/tests/Pagination.spec.js

```diff
@@ -114,5 +105,25 @@ describe("Pagination.vue", () => {
     });
 
 
+    it('Has a paginate() method that sets the current page in the URL query', () => {
```

Comment:
You cheat here but setting up wrapper.vm.currentPage manually.

Reply:
what should I do?

Reply:
I fixed it by adding a isTesting variable in the root and make sure in testing it will make sure is allowed for paginate